conf/nautilus/ansible/sanity_5node_cluster.yaml

```diff
@@ -0,0 +1,33 @@
+globals:
```

Comment:
Should we add more nodes & roles to cover other core areas? For example, rbd, cephfs... Also, would it make sense to rename the file to `tier_0_deploy.yaml`